homeassistant/components/zwave_js/discovery.py

```diff
@@ -579,8 +597,19 @@ def async_discover_values(node: ZwaveNode) -> Generator[ZwaveDiscoveryInfo, None
             ):
                 continue
 
-            # check label
-            if schema.label is not None and value.node.label not in schema.label:
+            # check firmware_version_range
+            if schema.firmware_version_range is not None and (
+                (
+                    schema.firmware_version_range.min is not None
+                    and AwesomeVersion(schema.firmware_version_range.min)
```

Comment:
Should we use `AwesomeVersion` already in the schema dataclass attribute, to avoid creating that instance multiple times?